info.xml

```diff
@@ -7,11 +7,11 @@
     <author>Compucorp</author>
     <email>info@compucorp.co.uk</email>
   </maintainer>
-  <releaseDate>2020-11-12</releaseDate>
-  <version>1.1.0</version>
+  <releaseDate>2021-03-25</releaseDate>
+  <version>1.2.0</version>
   <develStage>stable</develStage>
   <compatibility>
-    <ver>5.24</ver>
+    <ver>5.35</ver>
```

Comment:
not 5.35.1?

Reply:
civicrm ver does not support the 3 parts of the version, that's why we add the full version in the comments part (which I just added)